src/component/header/index.tsx

```diff
@@ -42,11 +43,28 @@ function Header(props: HeaderProps) {
     };
   }, [profileDropdown, setProfileDropdown]);
 
+  const navigate = useNavigate();
+
   return (
     <PlasmicHeader
       {...rest}
       profileDropdown={profileDropdown}
       toggleMobileSidebar={{ onClick: () => setIsVisible(mobileSidebar) }}
+      svg={{
+        wrap(node) {
+          return (
+            <a
```

Comment:
Realized a `Link` from `react-router-dom` makes more sense here. I'll do it in the next PR.